crates/biome_html_formatter/src/html/auxiliary/opening_element.rs

```diff
@@ -12,11 +30,15 @@ impl FormatNodeRule<HtmlOpeningElement> for FormatHtmlOpeningElement {
             r_angle_token,
         } = node.as_fields();
 
-        write!(f, [l_angle_token.format(), name.format(),])?;
+        write!(f, [l_angle_token.format(), name.format()])?;
         if attributes.len() > 0 {
             write!(f, [space(), attributes.format()])?
         }
-        write!(f, [r_angle_token.format()])?;
+        if !self.r_angle_is_borrowed {
+            write!(f, [r_angle_token.format()])?;
+        } else {
+            // write!(f, [r_angle_token.format()])?;
```

Comment:
Should we just drop the `else`? The commented line is the same as the one in the other branch, so it doesn't describe or preserve anything.

Reply:
Yes, I meant to drop the else branch.